CHANGELOG.md

```diff
@@ -327,17 +324,12 @@ if your ONI instance has a lot of custom code.
       code](https://github.com/open-oni/open-oni/commit/c40ca4eb87a47865c1d88cb92d3b9d316cea277f)
     - [Convert `optparse` to `argparse` in custom management
       commands](https://github.com/open-oni/open-oni/commit/37854ea135ce011add09040de06506c794e130d0)
-- Enable Sass support
```

Comment:
Karin says we can worry about that on our end, carry on

Reply:
Yeah, the loss of SASS wasn't something I expected (though I generally do prefer fewer production dependencies).  When the Py 3 stuff is figured out we can look at this again.  I may tinker with it some more soon, even, just because I hate unsolved problems.